realm/realm-jni/src/io_realm_internal_Group.cpp

```diff
@@ -300,17 +275,6 @@ JNIEXPORT jstring JNICALL Java_io_realm_internal_Group_nativeToString(
     return 0;
 }
 
-JNIEXPORT jboolean JNICALL Java_io_realm_internal_Group_nativeEquals(
```

Comment:
Is this useless? Seems more like `Group.equals()` should actually use it instead?